nomad/api/jobs.py

```diff
@@ -62,24 +63,35 @@ def __iter__(self):
         jobs = self.get_jobs()
         return iter(jobs)
 
-    def get_jobs(self, prefix=None, namespace=None):
-        """ Lists all the jobs registered with Nomad.
-
-           https://www.nomadproject.io/docs/http/jobs.html
-            arguments:
-              - prefix :(str) optional, specifies a string to filter jobs on based on an prefix.
-                        This is specified as a querystring parameter.
-              - namespace :(str) optional, specifies the target namespace. Specifying * would return all jobs.
-                        This is specified as a querystring parameter.
-            returns: list
-            raises:
-              - nomad.api.exceptions.BaseNomadException
-              - nomad.api.exceptions.URLNotFoundNomadException
+    def get_jobs(  # pylint: disable=redefined-builtin
+        self,
+        prefix: Optional[str] = None,
+        namespace: Optional[str] = None,
+        filter: Optional[str] = None,
```

Comment:
My suggestion is: Let's use `_filter` here instead of builtin value.

I think it's better to be slightly different from the official API than shadow keywords.